code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -84,6 +84,8 @@
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
 		H.apply_damage(drill_damage, BRUTE, "chest")
+		if(target.stat == DEAD)
+			target.gib()
 	else if(target.stat == DEAD && target.butcher_results)
 		target.harvest(chassis) // Butcher the mob with our drill.
```

Comment:
Makes this code unreachable and probably stops mechs from butchering butcherable mobs.